easybuild/easyconfigs/g/GAMESS-US/compddi.patch

```diff
@@ -0,0 +1,13 @@
+--- gamess.orig/ddi/compddi	2013-05-04 08:14:15.000000000 +1200
```

Comment:
always add a comment on top of patches about what it does, why it is needed and who made it :wink:

This is to avoid to ending up with patches that nobody really knows what they do.